apps/alpha-test-app/src/components/layout-demo/FloatingButtonDemo.vue

```diff
@@ -1,12 +1,12 @@
 <script lang="ts" setup>
 import chevronRightSmall from "@sit-onyx/icons/chevron-right-small.svg?raw";
-import { OnyxButton } from "sit-onyx";
+import { OnyxIconButton } from "sit-onyx";
 
 const isOpen = defineModel<boolean>();
 </script>
 
 <template>
-  <OnyxButton
+  <OnyxIconButton
```

Comment:
The icon button is basically not visible without hovering

Default:
![image](https://github.com/SchwarzIT/onyx/assets/67898185/6f135447-5b26-4e77-9956-f98ca1e1f0e2)

Hover:
![image](https://github.com/SchwarzIT/onyx/assets/67898185/068dc824-eefc-49ee-b256-07bedcaf514d)